dataactvalidator/config/awardFinancialRules.csv

```diff
@@ -1,16 +1,16 @@
 rule_type,is_single_field,field_name,rule_text_one,rule_text_two,description,rule_timing,rule_label

```

Comment:
The order of the `rule_timing` and `rule_label` in this file doesn't match the order in appropRules.csv.